vars/other.groovy

```diff
@@ -1,4 +1,30 @@
-def map =
+def get_current_stage(start_stage, only_stage){
+  def stages = ['Create VMs':true,
+               'Install Nmon':true,
+               'Add extra providers':true,
+               'Set RHV provider concurrent VM migration max':true,
+               'Conversion hosts enable':true,
+               'Configure oVirt conversion hosts':true,
+               'Configure ESX hosts':true,
+               'Create transformation mappings':true,
+               'Create transformation plans':true,
+               'Start performance monitoring':true,
+               'Execute transformation plans':true,
+               'Monitor transformation plans':true,
+               'Stop performance monitoring':true]
+
+  for (e in stages) {
+    if (e.key == start_stage ) {
```

Comment:
The following two if lines can be combined into one negating the condition and executing break.
Like:
```
if (e.key == start_stage && !only_stage)
    break;
```

Reply:
Your solution is wrong in case when only_stage=True .

Reply:
Your solution is wrong in case when only_stage=True.
I will leave it as is.